novice-goals.Rmd

```diff
@@ -0,0 +1,227 @@
+# Outline {#outline}
+
+This outline describes the questions that the novice courses on R and Python will answer.
+The advanced course can then assume that learners have hands-on experience with these topics
+but nothing more.
+
+## Getting started
```

Comment:
@gvwilson I would put the brief intro to computation as the first section and it would include the following:

-   Humans communicate with computers by modulating flows of current to instruct the computer to carry out tasks that are to complex to do by hand.
-   Transition from moving physical wires to graphical point and click interfaces.
-   Programming languages are formal languages, just like mathematics. This makes them effective and with less ambiguity than natural languages.
-   Communicating via typing text might seem more primitive than pint and click graphics, but has many advantages. Compare to using a dictionary to learn a language.
-  Conceptual understanding example of graphical vs text: Instead of using the mouse to click the "File" menu, then on "Open file", and then find the filename and click it, we would type `file.open('filename')`.

I am afraid this is a bit wordier than the bullets in the rest of the outline, I just wanted to ensure it is clear. I can draft the first version of this when we get into writing the content.